src/scripts/components/elements/validation.js

```diff
@@ -66,4 +73,49 @@ export default class Validation {
       'h5p-reader-question-required-wrapper', 'h5p-reader-question-hidden'
     );
   }
+
+  /**
+   * Validate input/editor content
+   * @param {number} contentLength length of content
+   * @returns {boolean}.
+   */
+  validate(contentLength) {
+    const remainingChars = this.params.charactersLimit - contentLength;
+    this.params.fieldContainer.classList.remove('validation-error');
+    let isError = this.validateCharsLimit(contentLength) ? true : false;
+
+    if (this.params.isRequired && contentLength === 0) {
+      this.params.fieldContainer.classList.add('validation-error');
+      this.showError(this.params.i10n.requiredMessage);
+      isError = true;
+    }
+
+    return isError;
+  }
+
+  /**
+   * Validate input/editor content
+   * @param {number} contentLength length of content
+   * @returns {boolean}.
+   */
+  validateCharsLimit(contentLength) {
+    const remainingChars = this.params.charactersLimit - contentLength;
+    let isError = false;

```

Comment:
The variable `isError` is no really required here if there's just one condition it depends on. You can simple return `true` inside the comparison of `remainingChars < 0` and `false` afterwards.